src/frontend/Account.ts

```diff
@@ -297,24 +254,33 @@ class Account {
          * case - 4902 indicates that the chain has not been added to MetaMask.
          * @see https://docs.metamask.io/guide/rpc-api.html#usage-with-wallet-switchethereumchain
          */
-        await provider.send("wallet_addEthereumChain", [{ ...this._getNetworkParams() }]);
+        await _provider.send("wallet_addEthereumChain", [
+          { ...NETWORK_PARAMS[chainId], chainId: getChainIdFromRaw(chainId) },
```

Comment:
This actually means that if the user adds this network to their wallet, then they will always use our rpc url (when connected to the given network), even when not using our app.

We should definitely change this in the future.

Reply:
I can't find any suitable public RPC URLs for `ethereum` and `goerli`. But I changed the RPC URLs anyway. Let me know if you have more suitable RPC URLs to add to the metamask.

Reply:
The urls look good.